hello_world_on_chain/test/user.ts

```diff
@@ -1,79 +1,92 @@
+// noinspection DuplicatedCode
+
 import hre from "hardhat";
 import {loadFixture} from "@nomicfoundation/hardhat-toolbox/network-helpers";
 import {expect} from "chai";
 import {ethers} from "ethers";
+import fs from "fs";
+import path from "path";
+import {User} from "../typechain-types";
 
-const fs = require("fs")
-const path = require("path")
+type EVMLinkData = {
+    publicKey: string;
+    transitiveAttestation: string;
+};
 
-function loadEVMLinkData(jsonPath: string) {
+function loadEVMLinkData(jsonPath: string): EVMLinkData {
     try {
-        const dir = path.resolve( __dirname, jsonPath);
-        const file = fs.readFileSync(dir);
+        const dir: string = path.resolve( __dirname, jsonPath);
+        const file: string = fs.readFileSync(dir, "utf8");
```

Comment:
I had to add the `"utf8"` argument after switching `fs` from `require` to `import`. Basically telling it to read in as a string because for whatever reason when we use `import` it defaults to bytes. Or it might be that it always defaulted to bytes but TypeScript couldn't perform analysis when `require` was being used